sdk/modelsrepository/Azure.Iot.ModelsRepository/src/ModelQuery.cs

```diff
@@ -4,7 +4,6 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Text.Json;
-using System.Threading.Tasks;
```

Comment:
Placeholder/reminder that there is a cleaner implementation of model parsing logic recently merged in the DMR tools repo, which I will update here after this PR is merged.